brian2/devices/cpp_standalone/templates/main.cpp

```diff
@@ -20,6 +20,37 @@
 
 #include <iostream>
 #include <fstream>
+#include <string>
+#include <sstream>
+
+
+std::string format_time(int time_in_s)
+{
+	int hours = time_in_s / 3600;
+	time_in_s = time_in_s % 3600;
+	int minutes = time_in_s / 60;
+	time_in_s = time_in_s % 60;
+
+	std::stringstream ss;
+	ss << hours;
+	std::string hrs = ss.str();
+
+	ss << minutes;
+	std::string mins = ss.str();
+	
+	ss << time_in_s;
+	std::string secs = ss.str();
+
+	if(hours < 1){
+		if(minutes < 1){
+			return secs + " s"
```

Comment:
Sorry, didn't see it before: there's a `;` missing at the end of the line here, making compilation fail.